kubetest/main.go

```diff
@@ -246,6 +248,20 @@ func getDeployer(o *options) (deployer, error) {
 		return dind.NewDeployer(o.kubecfg, o.dindImage, control)
 	case "gke":
 		return newGKE(o.provider, o.gcpProject, o.gcpZone, o.gcpRegion, o.gcpNetwork, o.gcpNodeImage, o.gcpImageFamily, o.gcpImageProject, o.cluster, &o.testArgs, &o.upgradeArgs)
+	case "eks":
+		cfg := eksconfig.NewDefault()
+		// configure via Prow presets
+		if err := cfg.UpdateFromEnvs(); err != nil {
+			return nil, err
+		}
+		// always overwrite with 'kubetest' configuration
+		cfg.KubetestControlTimeout = timeout
+		cfg.KubetestVerbose = verbose
+		// always let default log dumper handle artifacts
+		cfg.KubetestEnableDumpClusterLogs = false
```

Comment:
similar to above, I think we should respect the `--dump` flag, and skip dump if that's set to false